types/src/keyless/test_utils.rs

```diff
@@ -3,11 +3,12 @@
 use crate::{
     jwks::rsa::RSA_JWK,
     keyless::{
-        base64url_encode_bytes, base64url_encode_str,
+        base64url_encode_str,
         circuit_testcases::{
             SAMPLE_EPK, SAMPLE_EPK_BLINDER, SAMPLE_ESK, SAMPLE_EXP_DATE, SAMPLE_EXP_HORIZON_SECS,
             SAMPLE_JWK, SAMPLE_JWK_SK, SAMPLE_JWT_EXTRA_FIELD, SAMPLE_JWT_HEADER_B64,
-            SAMPLE_JWT_PARSED, SAMPLE_PEPPER, SAMPLE_PK, SAMPLE_PROOF, SAMPLE_UID_KEY,
+            SAMPLE_JWT_HEADER_DECODED, SAMPLE_JWT_PARSED, SAMPLE_JWT_PAYLOAD_DECODED,
```

Comment:
I prefer _JSON to maintain the 

_B64 vs _JSON 

decoded lil ambiguous to me

for HEADER and PAYLOAD

Reply:
Again, will fix this later, since we need to merge to release v1.10

Reply:
Fixed now.